Model/ReceiptDataProvider.php

```diff
@@ -206,7 +212,8 @@ public function __construct(
         OrderInterface $orderInterface,
         transactionBuilder $transactionBuilder,
         Config $gatewayConfig,
-        ApiData $apiData
+        ApiData $apiData,
+        LoggerInterface $logger
```

Comment:
di compile going through with fine ?
context might have already instance of logger interface.
$context->getLogger(); // return \Psr\Log\LoggerInterface

Reply:
the context does not seem to have the logger, di compile is going through ok